src/sentry/receivers/outbox/control.py

```diff
@@ -62,3 +65,35 @@ def process_organization_integration_update(object_identifier: int, **kwds: Any)
     ) is None:
         return
     organization_integration  # Currently we do not sync any other organization integration changes, but if we did, you can use this variable.
+
+
+@receiver(process_control_outbox, sender=OutboxCategory.WEBHOOK_PROXY)
+def process_async_webhooks(payload: Any, region_name: str, **kwds: Any):
+    from sentry.models.outbox import ControlOutbox
+    from sentry.silo.client import RegionSiloClient
+    from sentry.types.region import RegionResolutionError, get_region_by_name
+
+    try:
+        region = get_region_by_name(name=region_name)
+    except RegionResolutionError as e:
+        logger.error("webhook_proxy.region_resolution_error", extra={"error": e})
+        return
+
+    try:
+        webhook_payload = ControlOutbox.get_webhook_payload_from_outbox(payload=payload)
+    except Exception as e:
+        logger.error("webhook_proxy.invalid_payload_error", extra={"error": e})
+        return
```

Comment:
logger.error will report to Sentry, right?

Reply:
Your inclination here is a good one, but does make resolving issues difficult. I don't think we have a dead letter queue implementation, but that would be the canonical way to handle the case you're talking about.

Reply:
TIL about dead letter queues, that would be a more robust solution to implement across all the outboxes. I can keep this as is since yes, `logger.error` should create a sentry event.

Reply:
If we don't raise from this signal handler, the outbox push will be considered 'successful' and we'll delete the outbox message resulting in data loss. Other solutions would be to block on invalid data, or put into a dead-letter queue.

Reply:
I'm a little concerned about the potential data loss here. Could we have this raise instead of silently fail. Raising will cause outboxes to backup until they can proceed again, but we also won't lose data.